invenio_rdm_records/records/jsonschemas/records/definitions-v1.2.0.json

```diff
@@ -27,8 +27,15 @@
       }
     ]
   },
-  "contributorType": {
```

Comment:
We should not edit an existing jsonschema. Always create a new version of the jsonschema once it has been released.

Reply:
Also, we'll need to add a migration from the previous roles to the new vocab roles, but likely that's for all vocabs we're changing.

Reply:
Thanks for pointing this out, I didn't check it but was lucky since Pablo already created a new version of the schemas for this release in this [commit](https://github.com/inveniosoftware/invenio-rdm-records/commit/97d27a3581e31670573e7a02753acefc3f29ae0c).

For the migration, should we add for each PR or do we do it at the end?